### PR TITLE
Feature 532: historic leaderboard data + round selector

### LINKED ISSUE #532: Leaderboard round switch

### PR DESCRIPTION
**Pull request by KinimodEtlon**
closes #532

### CODE REVIEW

**Review by Zuckerwattederivat — APPROVED** [review]

File: enums/RemainingFundingStrategy.enum.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export enum RemainingFundingStrategyEnum {
+  Burn = 'brun',
```
Comment (marhali) [inline enums/RemainingFundingStrategy.enum.ts:2]: nitpick(blocking): seems to be a typo?

File: components/app/leaderboard/LeaderboardDropdowns.vue [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    <button-emit-with-dropdown
+      button-name="Select Round"
+      class="rounded w-36 mt-10 border-white bg-white"
+      button-style="text-third"
+      label-style="-m-3 text-white"
+      :menu-items="roundItems"
+      @selected="setItems"
+    />
+  </div>
+</template>
```
Comment (marhali) [inline components/app/leaderboard/LeaderboardDropdowns.vue:12]: nitpick(blocking): we should use singular in the component name here

File: components/app/leaderboard/LeaderboardDropdowns.vue [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+<script>
+import ButtonEmitWithDropdown from '@/components/common/ButtonEmitWithDropdown.vue';
+
+export default {
+  name: 'LeaderBoardDropdowns',
```
Comment (marhali) [inline components/app/leaderboard/LeaderboardDropdowns.vue:18]: nitpick(blocking): we used **Leaderboard** before, Keep the previous naming style

**Review by marhali — CHANGES_REQUESTED** [review]
Just a few small tasks :)

**Review by marhali — APPROVED** [review]

**Review by Abrom8 — APPROVED** [review]

**Pull request merged**